### PR TITLE
Javascript javascript2 week3

### PR DESCRIPTION
**Pull request by NataliaS1994**
ready to review

### CODE REVIEW

File: javascript/javascript2/week3/index.html [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <button id="click-button">Click</button> <br>
+    <button id="location"> Log location</button><br>
+    <p id="yourLocation"></p>
+     
+
```
Comment (MirSzabo) [inline javascript/javascript2/week3/index.html:14]: do not leave empty spaces without reason
use formatting plugins to format the code

File: javascript/javascript2/week3/warmup.js [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+    }
+
+    else {
+        logBadJoke()
+    }
+}
+
+jokeCreator(true, logFunnyJoke, logBadJoke);
+jokeCreator(false, logFunnyJoke, logBadJoke); 
+
```
Comment (MirSzabo) [inline javascript/javascript2/week3/warmup.js:103]: nice work! 👍

File: javascript/javascript2/week3/fastestPresser.js [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+        document.removeEventListener("keydown", keyUser)
+    }, timer * 1000);
+});
+
+//keypressed
+let keyUser = (event) => {
+    let keyChar = event.key
+    if (keyChar === "s") {
+        countS++;
+        showActive(".s-container");
```
Comment (MirSzabo) [inline javascript/javascript2/week3/fastestPresser.js:43]: I'm getting error in console 
`Uncaught ReferenceError: showActive is not defined at HTMLDocument.keyUser `
and I don't see the counting for pressed button

File: javascript/javascript2/week3/fastestPresser.js [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+        sCount.innerHTML = countS;
+    } else if (keyChar === "l") {
+        countL++;
+        showActive(".l-container");
+        lCount.innerHTML = countL;
+    } else {
+        console.log("Another key was pressed");
```
Comment (MirSzabo) [inline javascript/javascript2/week3/fastestPresser.js:50]: would be better to display this on the page not on console.log

File: javascript/javascript2/week3/fastestPresser.js (hunk 3 above)
Comment (NataliaS1994) [reply javascript/javascript2/week3/fastestPresser.js:43]: I'm not getting an error, everything works well for me in the codepen. There is also the counting for the pressed button in the code below :)

File: javascript/javascript2/week3/index.html (hunk 1 above)
Comment (NataliaS1994) [reply javascript/javascript2/week3/index.html:14]: Sure, I will remember that next time :)

**Comment by NataliaS1994** [comment]
Thank you so much for your review, much appreciated! :)

**Pull request merged**